private/buf/bufgen/internal/bufgenplugin/plugin_config.go

```diff
@@ -117,7 +118,10 @@ func (c *binaryPluginConfig) PluginName() string {
 	if c.name != "" {
 		return c.name
 	}
-	return c.path[0]
+	if len(c.path) == 1 {
+		return c.path[0]
+	}
+	return stringutil.SliceToString(c.path)
```

Comment:
This update isn't strictly related to this PR, but this improves error reporting and in some other areas. For example, if a plugin is invoked by `go run github.com/example/cmd/protoc-gen-someplugin`, some error message will look like  `plugin [go, run, github.com/example/cmd/protoc-gen-someplugin] failed`, instead of `plugin go failed`